docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -79,14 +79,48 @@ function AppMenuExtensionPoint() {
 }
 ```
 
-#### Why does the extension have `onClick` and `path`?
+### Why does the extension have `onClick` and `path`?
 
 Each extension link has either a `path` or an `onClick` property defined. There's never a scenario where both properties are defined at the same time.
 
 The reason for this behavior is that we want to be able to support both native browser links and callbacks. If the plugin adding the extension wants to navigate the user away from the current view into their app, then they can choose to define a path.
 
 If instead you want to open a modal or trigger a background task without sending the user away from the current page, then you can provide a callback.
 
+### Example: create an extension point for displaying components
```

Comment:
We have an example above with the link button without a similar header, we'll need to consider how to present these two different options as peers.